### PR TITLE
#28  Fix/open link position

### PR DESCRIPTION
**Pull request by vprovaggi**
Open button is now a link and positioned near the other commands.
The message is also reusing the feedback panel, in line with the other feedback provided

### CODE REVIEW

File: ui/src/alfresco/DockerContainerCreate.tsx [hunk 1]
```diff
@@ -90,6 +97,22 @@ const CommandPanel = ({ alfrescoState, commands }) => {
...
         </Button>
+        {isReady(alfrescoState) ? (
+          <Link
+            onClick={(e) => {
+              e.preventDefault();
+              openAlfrescoInBrowser();
+            }}
+            href="#"
+            variant="h3"
+            //color={colors.green[500]}
```
Comment (aborroy) [inline ui/src/alfresco/DockerContainerCreate.tsx:108]: Why is this dead code left commented?

File: ui/src/alfresco/DockerContainerCreate.tsx [hunk 2]
```diff
@@ -111,27 +134,37 @@ const FeedbackPanel = ({ alfrescoState }) => {
     message = 'Stopping Alfresco containers...';
     actionInProgress = true;
   }
+  if (isReady(alfrescoState)) {
+    message =
+      "Alfresco is Ready! Click the 'Open in browser' link and use the default credentials admin/admin";
```
Comment (aborroy) [inline ui/src/alfresco/DockerContainerCreate.tsx:139]: We may add this as i18n resource, despite we're accepting only English as language currently.

**Review by aborroy — APPROVED** [review]
Minor changes, looks ready to be merged.

File: ui/src/alfresco/DockerContainerCreate.tsx (hunk 2 above)
Comment (vprovaggi) [reply ui/src/alfresco/DockerContainerCreate.tsx:138]: Opened an issue to refactor the messaging.

File: ui/src/alfresco/DockerContainerCreate.tsx (hunk 1 above)
Comment (vprovaggi) [reply ui/src/alfresco/DockerContainerCreate.tsx]: good catch, removed

**Pull request merged**